Vagrantfile

```diff
@@ -53,7 +53,7 @@ ora_mount_dir = "edx-ora2"
 
 if ENV['VAGRANT_MOUNT_BASE']
 
-  ora2_mount_dir = ENV['VAGRANT_MOUNT_BASE'] + "/" + ora2_mount_dir
+  ora_mount_dir = ENV['VAGRANT_MOUNT_BASE'] + "/" + ora_mount_dir
```

Comment:
I assume this change it to match the variable name at L52? It may be better to just make this in a separate PR because it's a separate change, and will go in quickly.